indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -419,6 +419,9 @@ def get_track_reviewing_state(self, track):
         else:
             return AbstractReviewingState.mixed
 
+    def get_track_reviews(self, track):
```

Comment:
no strong opinion, but for a simple list comprehension in 2 places i'd probably keep the list comprehensions.